docs/fields-and-widgets.rst

```diff
@@ -146,7 +146,7 @@ Form fields
 ``JSONField``
 ~~~~~~~~~~~~~
```

Comment:
Please, change this heading as well:

```suggestion
``JSONFormField``
~~~~~~~~~~~~~~~~~
```

Reply:
done, thank you for making it a suggestion, it was painless to do.